art/experiment/experiment_state.py

```diff
@@ -12,6 +11,7 @@ class ArtProjectState:
     step_states: Dict[str, Dict[str, Dict[str, str]]]
     status: str
     """
+    Singleton that knows the current experiment state + have information about the world around
```

Comment:
It is not a singletone

Reply:
Ofc, it was just a leftover which I dummy copy passed into the doc string.